Polyhedron/demo/Polyhedron/CGAL_double_edit.cpp

```diff
@@ -9,13 +9,14 @@ class DoubleValidator : public QDoubleValidator
     : QDoubleValidator(parent)
   {
     setLocale(QLocale::C);
+    setNotation(QDoubleValidator::StandardNotation);
   }
 
   void fixup ( QString & input ) const
   {
     input.replace(".", locale().decimalPoint());
     input.replace(",", locale().decimalPoint());
-    QDoubleValidator::fixup(input);
+//    QDoubleValidator::fixup(input);
```

Comment:
`fixup` should call its base `fixup` implementation. If that does not work, then it means that the setting of the validator is wrong. Maybe the range?